presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -2,10 +2,23 @@
 JDBC Driver
 ===========
 
-Presto can be accessed from Java using the JDBC driver.
```

Comment:
I feel like the original sentence says the same thing in a simpler way. There doesn't seem to be any value in defining the term "JDBC" as the "from Java" part tells users everything they need to know.